independent-projects/tools/registry-client/src/main/java/io/quarkus/registry/util/PlatformArtifacts.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkus.registry.util;
+
+import io.quarkus.maven.ArtifactCoords;
+import io.quarkus.registry.Constants;
+
+public class PlatformArtifacts {
+
+    public static ArtifactCoords bomToCatalog(ArtifactCoords bom) {
+        return bomToCatalog(bom.getGroupId(), bom.getArtifactId(), bom.getVersion());
+    }
+
+    public static ArtifactCoords bomToCatalog(String bomGroupId, String bomArtifactId, String bomVersion) {
```

Comment:
```suggestion
    public static ArtifactCoords getCatalogCoordsFromBom(String bomGroupId, String bomArtifactId, String bomVersion) {
```

Reply:
If we do that, shouldn't we go all the way? PlatformArtifacts.getCatalogArtifactFromBomArtifact(...)?

Reply:
I wouldn't mind, as soon as it's descriptive enough. 

My point is that with `bomToCatalog` you need to look at the signature and the code to understand what it does.